### DISCUSSION TITLE
Extension to TF-card port, will it work?

### DISCUSSION THREAD

**Discussion by steve8x8** [OpenNeptune3D/OpenNept4une#91] [Q&A]
To better access the TF card port on the mainboard (the one that is only required for MCU firmware flashing), without moving the printer or cutting the front panel open, I'd like to use a TF extender.
In another context (Tonie box) it was necessary to remove a terminating resistor on the "far side", right next to the socket for the card - will I have to keep that one for the N4, or will I have to follow the same removal procedure? Is it safe at all, iow, is access speed an issue?

**Comment by halfmanbear** [comment]
I can't imagine there would be an issue with an extension. The data rates are very low, it's worth a try as is.
